### PR TITLE
Attempt at a calver monthly tag reset

### PR DESCRIPTION
**Pull request by lgray**
Would appreciate commentary here on if this is a good idea. :-)

### CODE REVIEW

File: .github/workflows/month_tag.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        with:
+          fetch-depth: 0
+      - name: format calver prefix
+        id: prefix
+        run: |
+          echo "::set-output name=date::$(date +'%Y.%-m')"
+      - name: extract tag message
+        id: tagmessage
+        run: |
+          echo "::set-output name=tagmessage::$(git log --pretty='format: - %s' --first-parent $(git describe --tags --abbrev=0)..HEAD)"
```
Comment (matthewfeickert) [inline .github/workflows/month_tag.yml]:
```suggestion
          echo "tagmessage=$(git log --pretty='format: - %s' --first-parent $(git describe --tags --abbrev=0)..HEAD)" >> $GITHUB_OUTPUT
```

`set-output` commands via stdout is deprecated and will start failing soon (https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/) so use the new approach of appending to the [`$GITHUB_OUTPUT` environment file](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#environment-files).

c.f. https://github.com/scikit-hep/pyhf/pull/2155 for further examples.

**Review by matthewfeickert — CHANGES_REQUESTED** [review]

File: .github/workflows/month_tag.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    name: monthly release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: format calver prefix
+        id: prefix
+        run: |
+          echo "::set-output name=date::$(date +'%Y.%-m')"
```
Comment (matthewfeickert) [inline .github/workflows/month_tag.yml]:
```suggestion
          echo "date=$(date +'%Y.%-m')" >> $GITHUB_OUTPUT
```

Missed this one.

**Review by matthewfeickert — CHANGES_REQUESTED** [review]

**Comment by jpivarski** [comment]
Now would be a very good time to switch from semantic versions to calendar versions, if you prefer calendar versions.

But... _automatic_ releases? Something that happens once a month is not a lot of manual burden, and while a human in the loop doesn't guarantee that it's bug-free, it's a safety measure that prevents obviously bad things from happening. What about automated reminders and a one-button procedure instead?

Then again, maybe more users of Coffea pin versions, since they're at the end of a dependency chain (applications, not libraries).

Maybe my question is, what projects actually do this?

**Comment by lgray** [comment]
@jpivarski You make some good points - an automated reminder may be a better idea. Though I find even with those I'll end up not doing something.

I'd kind of prefer an automatic monthly release, just to deal with that mode of failure. 

Even better would be something that makes a release if there are commits since the last tag.  This would deal better with the random failures I am trying to avoid with this.

**Comment by lgray** [comment]
obviated by #813

**Pull request closed without merging**